### PR TITLE
documentation(EJ2-64147): Prepare the predefined dialog UG documentat…

### PR DESCRIPTION
**Pull request by selvakumar-3626**

### CODE REVIEW

**Review by thangavele — APPROVED** [review]

File: ej2-asp-core-mvc/code-snippet/predefined-dialogs/dimension/alert-min-height-width/taghelper [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<style>
+    .e-dialog.customClass {
+        min-width: 300px;
+        min-height: 200px;
+    }
+</style>
+<script>
+    document.getElementById('alertBtn').onclick = function () {
+        ej.popups.DialogUtility.alert({
+            title: 'About SYNCFUSION Succinctly Serires',
```
Comment (thangavele) [inline ej2-asp-core-mvc/code-snippet/predefined-dialogs/dimension/alert-min-height-width/taghelper:13]: Serires - is it correct here?

**Comment by selvakumar-3626** [comment]
rebuild

**Comment by selvakumar-3626** [comment]
rebuild

**Review by thangavele — APPROVED** [review]

**Review by thangavele — APPROVED** [review]

File: ej2-asp-core-mvc/predefined-dialogs/EJ2_ASP.MVC/animation.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+description: Checkout and learn about animation in ##Platform_Name## Predefined Dialogs of Syncfusion Essential JS 2 and more details.
+platform: ej2-asp-core-mvc
+control: Animation
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# Animation in Predefined Dialogs 
+
+The predefined dialogs can be animated during the open and close actions. You can customize the `Delay`, `Duration`, and `Effect` of animation by using the `animationSettings` property.
```
Comment (saravanangn) [inline ej2-asp-core-mvc/predefined-dialogs/EJ2_ASP.MVC/animation.md:13]: change the casing for `Delay`, `Duration`, and `Effect`

**Pull request merged**